docs/change_controlled.rst

```diff
@@ -0,0 +1,82 @@
+###################################
+Writing Change-Controlled Documents
+###################################
+
+Change-controlled documents are documents that require approval from a Change Control Board (CCB) before they can be released.
+The project-wide change control process is described in `LPM-19`_, with the process for managing documents being described in `LPM-51`_, and covers those documents using DocuShare handles "LSE" or "LPM" that must be approved by the Project `CCB`_.
+For Data Management, these documents have the DocuShare handle "LDM", and they must be approved by the DM CCB using the :ref:`RFC <decision-making-rfc>` process.
+These change-controlled documents can be written using the Word templates defined in `Document-9224`_, but the preference within Data Management is to write change-controlled documents in LaTeX using the ``lsstdoc`` document class provided by the `lsst-texmf`_ package and to develop these documents using ``git`` repositories within the LSST organization on `GitHub`_.
```

Comment:
I _personally_ have a preference for LaTeX over Word, but this is the first time I recall seeing it expressed in official DM documentation — my understanding from e.g. LDM-294 is that the latter is a first class citizen. Do we really want to change that in this document?

Reply:
So... *which* of "these change-controlled documents can be written..." etc? Is the process defined in this document designed to be used for all of LPM, LSE, LDM, and others, or just for DM-level change control? Will we publish LPMs at `lpm-xxx.lsst.io`, for example?